iquip/apps/scheduler.py

```diff
@@ -91,34 +87,27 @@ def updateInfo(self, info: Optional[ExperimentInfo] = None):
         _dismiss_items(self.argsLayout)
         self.experimentInfo = info
         if info is not None:
-            self.nameLabel.setText(info.name)
-            for key, value in info.arginfo.items():
+            for key, value in info.items():
                 if key != "priority":
                     self.argsLayout.addWidget(QLabel(f"{key}: {value}", self))
         else:
             self.nameLabel.setText("None")
 
 
 class ExperimentView(QWidget):
-    """Widget for displaying the information of the experiment.
-    
-    Attributes:
-        nameLabel: The QLabel instance for displaying the experiment name.
-    """
+    """Widget for displaying the information of the experiment."""
 
-    def __init__(self, info: ExperimentInfo, parent: Optional[QWidget] = None):
+    def __init__(self, info: SubmittedExperimentInfo, parent: Optional[QWidget] = None):
         """Extended.
         
         Args:
             info: The information of the experiment.
         """
         super().__init__(parent=parent)
         # widgets
-        self.nameLabel = QLabel(info.name, self)
-        labels = (QLabel(f"{key}: {value}", self) for key, value in info.arginfo.items())
+        labels = tuple(QLabel(f"{key}: {value}", self) for key, value in info.items())
```

Comment:
Is `tuple()` necessary?

Reply:
If not, the parantheses are just an envelope that cases `generator`-typed value. Thus, the type of `labels` would be `<class 'generator'>`.